app/controllers/providers/no_national_insurance_numbers_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Providers
+  class NoNationalInsuranceNumbersController < ProviderBaseController
+    include ApplicantDetailsCheckable
+
+    def show
+      @applicant = legal_aid_application.applicant
+
+      if @applicant.national_insurance_number?
+        redirect_to providers_legal_aid_application_check_benefits_path(legal_aid_application)
+      else
+        details_checked! unless details_checked?
+        legal_aid_application.create_benefit_check_result!(result: "skipped:no_national_insurance_number")
```

Comment:
What does this mean?

Reply:
had a technical chat with Rose about this and other approaches the other day and settled on this one. It gives the benefit result object a negative response but with some information. This is also done in the check benefits controller for "known_issues"

Reply:
would it clarify things if extracted to a method as below or is your question more existential?

```
def mark_as_benefit_check_skipped!
  legal_aid_application.create_benefit_check_result!(result: "skipped:no_national_insurance_number")
end
```

Reply:
Yeah, a more descriptive method would be helpful 👍